native_client/deepspeech.cc

```diff
@@ -103,28 +124,66 @@ Model::infer(float* aMfcc, int aNFrames, int aFrameLen)
   std::vector<Tensor> outputs;
   Status status = mPriv->session->Run(
     {{ "input_node", input }, { "input_lengths", n_frames }},
-    {"output_node"}, {}, &outputs);
+    {"Reshape_3"}, {}, &outputs);
```

Comment:
We should name that node better ; in my code I already did it there: https://github.com/mozilla/DeepSpeech/pull/830/commits/65b4768202dac67c5cdf3d3eb7ce7d155a83fa11#diff-79c8372f97f7f5a6d8660bd265f836d7R461

Hence we have a nicer and more meaningful `logits_output_node` name :-)

Reply:
Yes, the problem is that this breaks compatibility with older models. I don't see a good way to solve both cases, I guess I could try to use the new name and if it fails use the existing name, but that's kind of ugly.

Reply:
Yep, I suspected so, but since we have not yet any official model, I thought now would be a good time to rename the tensor for a more meaningful name :)